pom.xml

```diff
@@ -208,6 +208,13 @@
       <scope>test</scope>
     </dependency>
 
+    <dependency>
+      <groupId>org.slf4j</groupId>
+      <artifactId>log4j-over-slf4j</artifactId>
```

Comment:
this isn't quite what meant, actually. I meant [slf4j-log4j12](slf4j-log4j12).

`log4j-over-slf4j` goes the other way, making log4j calls turn into slf4j calls.